src/Disks/ObjectStorages/S3/S3ObjectStorage.cpp

```diff
@@ -148,7 +148,8 @@ class S3IteratorAsync final : public IObjectStorageIteratorAsync
             auto objects = outcome.GetResult().GetContents();
             for (const auto & object : objects)
             {
-                ObjectMetadata metadata{static_cast<uint64_t>(object.GetSize()), Poco::Timestamp::fromEpochTime(object.GetLastModified().Seconds()), object.GetETag(), {}};
+                /// FIXME: support tags via getObjectTagging()
```

Comment:
Note, that this will not work for glob, yet

```sql
azat.local$  :) select _tags, _file from s3(s3_conn, filename='*') limit 1

SELECT
    _tags,
    _file
FROM s3(s3_conn, filename = '*')
LIMIT 1

Query id: c91c08da-810c-404f-a398-dcf81a898035

   ┌─_tags─┬─_file─┐
1. │ {}    │ tags  │
   └───────┴───────┘

1 row in set. Elapsed: 0.022 sec.

azat.local$  :) select _tags, _file from s3(s3_conn, filename='tags') limit 1

SELECT
    _tags,
    _file
FROM s3(s3_conn, filename = 'tags')
LIMIT 1

Query id: 301e9ff8-e8ce-4e16-87aa-933967f57f55

   ┌─_tags─────────┬─_file─┐
1. │ {'foo':'bar'} │ tags  │
   └───────────────┴───────┘

```